src/core/migrations/0102_merge_20250217_0954.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 4.2.18 on 2025-02-17 09:54
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0100_contact_ip_address_null'),
+        ('core', '0101_delete_blank_keywords'),
+    ]
```

Comment:
These migrations shouldn't conflict, I believe the solution is to address their dependencies

Reply:
Sure, happy to do that. My understanding has been that if a migration is already merged, don't change its dependencies. Sounds like that's not always the case?

Reply:
This conflict arises from changing the dependencies in this PR so it is worth setting them right. Not so sure how the other happened...This conflict arises from changing the dependencies in this PR so it is worth setting them right. Not so sure how the other happened...